plugins/menu.js

```
@@ -1,22 +1,24 @@
 import { font } from '#bot';
 import { config } from '#config';
-import { bot, commands, getConfigValues } from '#lib';
+import { bot, commands } from '#lib';
+import { getConfig } from '#sql';
 import { formatBytes, runtime } from '#utils';
 import { platform, totalmem, freemem } from 'os';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)